### PR TITLE
Memefixes 3

### LINKED ISSUE #966: Tauren Totem it's a weapon, but it's not
**Mod Version**
6ca57710

**What expansions do you have installed?**
All

**Are you using any submods/mods? If so, which?**
No

**Please explain your issue in as much detail as possible:**
Tauren Totem have combat_weapon flag, but it's not a weapon

**Steps to reproduce the issue:**
Duel with equipped tauren totem

**Upload an attachment below: .zip of your save, or screenshots:**

<details>
<summary>Click to expand</summary>

image

image

</details>

### LINKED ISSUE #1188: Zem'lan's Link
**Describe your suggestion in full detail below:**
Add gamepedia link to Zem'lan.

### LINKED ISSUE #1201: Has a class in Artifact Conditions
**Mod Version**
0317ee1deb744cfa454aa5f410a1fb475d84b255

**Are you using any submods/mods? If so, which?**
No

**Please explain your issue in as much detail as possible:**
Localization is wrong. I have a class, but it returns false.

**Upload screenshots of the problem localization:**
<details>
<summary>Click to expand</summary>

image

</details>

### LINKED ISSUE #1216: [SUGGESTION] | Felo'Melorn changes
**Describe your suggestion in full detail below:**
I propose to replace the relationship with the cult of the sunwell with +5 to the attitude of same religious and give the opportunity to use all religions felomelorn. He does not reject the owner if he suddenly believes in the Old gods and even more so believes in the Vindicator.

### PR DESCRIPTION
**Pull request by MemeWizard**
#### Changelog:
- Added Zem'lan's gamepedia link. Fixes #1188.
- Tauren Totem is now a weapon. Fixes #966.
- Changed text of threat change tooltip from "Justified Invasion" to "Fearsome Conqueror", since neither the Scourge Invasion in Quel'Thalas and the First War are justified.
- Felo'melorn gives **Same Religion Opinion** instead of **Sunwell Opinion**. Closes #1216.

#### Developer changelog:
- Added `custom_tooltip` to `is_dragon_race_trigger`, `is_medium_humanoid_trigger` and `is_large_humanoid_trigger`
- Fixed #1201 and changed a lot of artifact active condition to generic trigger, since they are the same anyway.
- Implemented #1204. Additionally, now lowborn Antonidas and magocraties that get mage trait after game start can get Pipe of Insight.
- Changed default `Kama Sutra` artifact book icon to `A Steamy Romance Novel` artifact icon.

### CODE REVIEW

File: common/artifacts/901_wc_artifacts_weapon_twohanded.txt [hunk 1]
```diff
@@ -522,5 +458,29 @@ great_royal_sword = {
...
+	martial = 1
+	combat_rating = 10
+
+	monthly_character_prestige = 0.1
+	monthly_character_piety = 0.05
+
+	picture = "GFX_tauren_totem"
+
 	slot = weapon
 }
```
[the remark this answers is not in the document]
Comment (MemeWizard) [reply common/artifacts/901_wc_artifacts_weapon_twohanded.txt]: done

File: common/artifacts/901_wc_artifacts_weapon_twohanded.txt [hunk 2]
```diff
@@ -522,5 +458,26 @@ great_royal_sword = {
...
+}
+
+# Common
+tauren_totem = {
+	quality = 1
+	flags = { combat_weapon mace }
+
+	active = {
+		shared_physical_weapon_trigger = yes
+		culture_group = tauren_group
```
Comment (zumbak04) [inline common/artifacts/901_wc_artifacts_weapon_twohanded.txt]: I would make it for `is_large_humanoid_trigger` only.

Comment (MemeWizard) [reply common/artifacts/901_wc_artifacts_weapon_twohanded.txt]: done

**Comment by MemeWizard** [comment]
@zumbak04 thanks <3

**Pull request merged**